### ISSUE TITLE
Patrocínico para domínios

### ISSUE THREAD

**Issue by mathiasgrimm** [phpEmpregos/phpEmpregos.com#20]
Temos 2 domínios

phpempregos.com e phpempregos.com.br

Alguém tem alguma idéia de como conseguir esse patrocínio?

**Comment by gustavoper** [comment]
@mathiasgrimm  acredito inicialmente poderiamos pensar num rateio entre os colaboradores, uma vez que o pagamento é somente 1x ao ano. O mesmo vale pro hosting.

Ideal também seria um crowdfunding com um valor bem baixo, já que é um projeto sem fim lucrativo.

**Comment by dunice** [comment]
@mathiasgrimm concordo com a sugestão do @gustavoper . Acho que quando o projeto ganhar mais corpo, podemos colocar alguma integração para doações e/ou add-ons.
